### DISCUSSION TITLE
Updates to GOA grid 🗺

### DISCUSSION THREAD

**Discussion by MargaretSiple-NOAA** [afsc-gap-products/StationAllocationAIGOA#16] [General]
This is a question mostly for Zack -- if we allocate stations based on the new GOA survey design, do we have new stations that fall outside the existing GOA grid? As I understand it, the ragged edges of the existing grid are based on bathymetry, so the shape of those will change with the new design. But I don't know whether the new design has stations/grid cells outside the existing grid cell area, and if we do we may have to figure out how to make a new grid that covers the full extent.

Sean asked about this too; partly because he doesn't have an "original" grid file that covers the whole survey area (i.e., does not have those ragged edges) for the GOA. He says it should probably be the same old file for both the AI and the GOA. If we can snap the existing grid to the old, big, "original" one, we can extend it as needed with the new design, and eventually do the same if there is an AI redesign.

**Comment by zoyafuso-NOAA** [comment]
If we create the stations based on the bathymetry then we have the opposite problem: the footprint of the bathymetry raster doesn't fully creep to the edge of the GOA grid for some regions. See this [pdf](https://github.com/afsc-gap-products/StationAllocationAIGOA/files/9788773/bathy.pdf) of the AI/GOA bathy raster with the historical strata superimposed as an example. This means that for those grid cells we won't have bathy data to inform how to create those stations.  

For the western Shumagin part of the grid where the depth raster doesn’t extend far enough onto the existing GOA grid:

I looked through some of the plots in the 2018 EFH GOA GF report (Rooney et al. 2018 Tech Memo) because I imagine we are using similar bathy rasters and the analysis only extends to Unimak Pass (165 degrees longitude). The GOA survey extends farther west than the EFH GOA analysis, to about 170 degrees longitude. So that then led me to look at the 2017 EFH AI GF report (Turner et al. 2017 Tech Memo) which includes that western portion of the Shumagin region west of 165 degrees longitude and those depths only go down to 500 m. Is this because the AI survey only goes down to 500 m, so the AI depth raster only goes to 500 m? And because the GOA survey extends to 1000 m, the GOA depth raster goes down to 1000 m (or whatever the deepest depth is). That might be why don't see depths > 500 m at the very western tip of the grid.

**Comment by Ned-Laman-NOAA** [comment]
A couple of thoughts (not that this was addressed to me!!!) and a question.

1 - Zack is onto something here; we would've only needed depths to 500 m for the Aleutians. 
2 - Zimm's AI bathymetry extends much deeper than 500m (see M:\ZimmBathymetry\AI\ai_bathy) as do his wGOA & cGOA bathys (M:\ZimmBathymetry\wGOA\wgoa_bathy & M:\ZimmBathymetry\CGOA\cgoa_bathy)

Should we recreate the GOA bathymetry from the various sources and then re-stratify?

**Comment by zoyafuso-NOAA** [comment]
Okay good, that does make sense now. I'll look at those and see how much more of the grid is filled out.

**Comment by zoyafuso-NOAA** [comment]
This is what the strata and stations look like with the extended AI bathy map for just the Shumagin region. The areas in black are those not specified by the current bathy raster I got from the EFH analyses. ![](https://user-images.githubusercontent.com/60302777/196302886-d6329776-b4f8-4f45-a944-0da9f12e21b1.png)

When including the full AI bathy raster, I get this picture: 
![updated_strata](https://user-images.githubusercontent.com/60302777/196303149-d59e772c-2101-416b-a64c-cf0ddfe36330.png)

If you zoom in on it, you'll see that a lot of the western Shumagin area is now defined. There are still some stations "not defined" but these are actually areas that are > 1000 m, which I imagine exist in the current goa grid.

**Comment by Ned-Laman-NOAA** [comment]
That's a pretty dramatic change to the 1-100m stratum when you add the offshore bit that was missing!
Is this the latest untrawlable stations too?

**Reply by zoyafuso-NOAA** [reply]
Oh yeah, I didn't even see that! I was looking at the edge. I'll make sure to use the the rasters in the RACE_EFH_Variables folder for the comparison when I get access, it might be more consistent. I was using a dataset downloaded from the [NPRB](http://projects.nprb.org/#metadata/a90807da-6f92-4a49-9e1d-ff589d2f6dbd/project) for the offshore AI bit, it may not be the one used for the EFH layer. 

For the untrawlable areas, I took the goagrid2019_landuntrawlsndmn.shp shapefile and then added any new untrawlable station polygons from the latest pull of the GOA.GOAGRID_GIS table. So they should be current. Although what I didn't check was for instances where untrawlable stations in 2019 were switched to trawlable in 2021. Is that something that happens and would that be updated in the GOA.GOAGRID_GIS table (under the TRAWLABLE field)? If so, I can remove those now trawlable stations from the 2021 version.

**Comment by Ned-Laman-NOAA** [comment]
It isn't often, but Yes, untrawlables get changed to trawlables every year. The most up to date version of trawlable and untrawlable stations will be the latest pull from GOAGRID_GIS.

**Comment by zoyafuso-NOAA** [comment]
I'm gonna make the bathymetry discussion a separate topic because the bathymetry raster I'm using and the EFH raster are different for some reason.

**Comment by Ned-Laman-NOAA** [comment]
Not sure if this belongs here or in the "bathymetry discussion" that Zack referenced and I didn't see on the list of discussions here, but it hit me this morning that the discussions we were having yesterday about UT cells and and the rubric for applying UT knowledge to new strata we will have to have about T cells to a slightly lesser extent. We'll know where the good tow was and should be able to extend trawlability the new stratum that the good tow is now found in within it's grid cell, even to contiguous parts of that stratum within the same grid cell. Depth of the tow won't matter in the decision, just location of the tow within a stratum and grid cell, right?
